### DISCUSSION TITLE
还有哪些APP可以屏蔽更新？

### DISCUSSION THREAD

**Discussion by LuzMasonj** [blackmatrix7/ios_rule_script#265] [Q&A]
https://raw.githubusercontent.com/blackmatrix7/ios_rule_script/master/rewrite/QuantumultX/Upgrade/Upgrade.conf
最近把国内一些APP都抓包下载了旧版，但是打开APP时会弹出更新弹窗。自己尝试抓包，无奈水平太菜抓不到，除了以上链接中的APP，还有哪些APP可以屏蔽更新呢？希望能分享下。

**Comment by blackmatrix7** [comment]
目前这个只有我自己在维护，只有我看到哪个有更新提示，才会去抓包拦截，添加过程比较缓慢，主要满足我自己需求为主。

抓包我一般用Thor，查找链接里有类似check、update、upgrade、version之类的关键字，比较费时，你也可以参与进来一起维护。

**Reply by LuzMasonj** [reply]
> 目前这个只有我自己在维护，只有我看到哪个有更新提示，才会去抓包拦截，添加过程比较缓慢，主要满足我自己需求为主。
> 
> 抓包我一般用Thor，查找链接里有类似check、update、version之类的关键字，比较费时，你也可以参与进来一起维护。

我用QX调试模式 尝试对QQ抓包 结果毛都抓不到 联通营业厅倒是屏蔽了更新提醒 我要不提个pr？

**Reply by blackmatrix7** [reply]
> > 目前这个只有我自己在维护，只有我看到哪个有更新提示，才会去抓包拦截，添加过程比较缓慢，主要满足我自己需求为主。
> > 抓包我一般用Thor，查找链接里有类似check、update、version之类的关键字，比较费时，你也可以参与进来一起维护。
> 
> 我用QX调试模式 尝试对QQ抓包 结果毛都抓不到 联通营业厅倒是屏蔽了更新提醒 我要不提个pr？

可以啊，QQ这种IM一般都是会加密的，能抓包的可能性比较小，我没尝试过。

**Comment by blackmatrix7** [comment]
维护的话，可以提PR修改这个文件。

https://github.com/blackmatrix7/ios_rule_script/blob/master/source/rewrite/Upgrade/Upgrade.qxrewrite
